tuf/api/metadata.py

```diff
@@ -1379,13 +1373,13 @@ def from_data(
         return cls(length, hashes, target_file_path)
 
     def verify_length_and_hashes(self, data: Union[bytes, IO[bytes]]) -> None:
-        """Verifies that length and hashes of "data" match expected values.
+        """Verifies that length and hashes of ``data`` match expected values.
 
         Args:
-            data: File object or its content in bytes.
+            data: Target file object or its content in bytes.
 
         Raises:
-            LengthOrHashMismatchError: Calculated length or hashes do not
+            LengthOrHashMismatchError: The calculated length or hashes do not
```

Comment:
This one is probably left by mistake. Will revert it at the end when addressing other comments to save time.